charts/nats/values.yaml

```diff
@@ -188,13 +188,13 @@ reloader:
 exporter:
   enabled: true
   # resource configuration for the metrics sidecar
-  resources: {}
-  #  limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  #  requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+    limits:
+     cpu: 100m
+     memory: 128Mi
+    requests:
+     cpu: 100m
```

Comment:
Requests: CPU to 50m. memory to 100Mi
idea is not to be same as limits

Reply:
https://github.com/astronomer/astronomer/blob/master/values.yaml#L387-L399